README.md

```diff
@@ -1,2 +1,344 @@
 # next-intl
-Library to integrate react-intl with Next.js, with active locale management based on policies
+
+[![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
+
+[npm-url]:https://npmjs.org/package/@moxy/next-intl
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/next-intl.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/next-intl.svg
+[travis-url]:https://travis-ci.org/moxystudio/next-intl
+[travis-image]:https://img.shields.io/travis/moxystudio/next-intl/master.svg
+[codecov-url]:https://codecov.io/gh/moxystudio/next-intl
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/next-intl/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/next-intl
+[david-dm-image]:https://img.shields.io/david/moxystudio/next-intl.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/next-intl?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/next-intl.svg
+
+Library to integrate [react-intl](https://www.npmjs.com/package/react-intl) with Next.js, making it easy to manage the current locale based on configurable policies.
+
+
+## Installation
+
+```sh
+$ npm install --save @moxy/next-intl react-intl
+```
+
+If you are running Node.js `< 13.1.0`, you must also install `full-icu` and start node with [`--icu-data-dir=node_modules/full-icu`](https://github.com/zeit/next.js/blob/5e6f79117fae59ec3a6a3260808f611862c53f0a/examples/with-react-intl/package.json#L5).
+
+
+## Setup
+
+#### 1. Create a root folder named `intl` with the following structure:
+
+```
+intl/
+  index.js
+  messages/
+    en-US.json
+```
+
+The `index.js` file should have the following contents:
+
+```js
+import { cookiePolicy, acceptLanguagePolicy, defaultPolicy } from '@moxy/next-intl';
+
+export default {
+    locales: [
+        {
+            id: 'en-US',
+            name: 'English',
+            loadMessages: async () => {
+                const module = await import(/* webpackChunkName: "intl-messages/en-US" */ './messages/en-US.json');
+
+                return module.default;
+            },
+        },
+    ],
+    policies: [
+        cookiePolicy(),
+        acceptLanguagePolicy(),
+        defaultPolicy('en-US'),
+    ],
+};
+```
+
+You may declare more locales by adding them into the `locales` array.
+
+#### 2. Include `<NextIntlScript>` in `pages/_doc.js`:
```

Comment:
The correct path is `pages/_document.js`.

Reply:
Fixed!